build.gradle

```diff
@@ -302,8 +302,7 @@ dependencies {
     liquibaseRuntime 'org.postgresql:postgresql:42.2.7'
     liquibaseRuntime 'ch.qos.logback:logback-classic:1.2.3'
 
-    testImplementation group: 'junit', name: 'junit', version: '4.13.2'
-    testImplementation group: 'org.hamcrest', name: 'hamcrest', version: '2.1'
```

Comment:
We don't need dependencies on on junit or hamcrest since these are already included as part of the spring test dependency.